tests/test_detect_entities.py

```diff
@@ -35,3 +35,67 @@ def test_split_person():
     assert entities[1]["type"] == "PERSON"
     assert not entities[1]["in_source"]
     assert summary[entities[1]["start"] : entities[1]["end"]] == entities[1]["ent"]
+
+
+def test_split_person_apostrophe():
+    summary = "A search is under way for Daniel Levenson's cat"
+    source = "Pembrokeshire"
+    entities = detect_entities(summary, source)
+
+    assert len(entities) == 2
+
+    assert entities[0]["ent"] == "Daniel"
+    assert entities[0]["type"] == "PERSON"
+    assert not entities[0]["in_source"]
+    assert summary[entities[0]["start"] : entities[0]["end"]] == entities[0]["ent"]
+
+    assert entities[1]["ent"] == "Levenson's"
+    assert entities[1]["type"] == "PERSON"
+    assert not entities[1]["in_source"]
+    assert summary[entities[1]["start"] : entities[1]["end"]] == entities[1]["ent"]
+
+
+def test_split_person_quote():
+    summary = (
+        'John "Brad" Bradbury, drummer with The Specials, has died at the age of 62.'
+    )
+    source = "Pembrokeshire"
+    entities = detect_entities(summary, source)
+
+    assert len(entities) == 5
+
+    assert entities[0]["ent"] == "John"
+    assert entities[0]["type"] == "PERSON"
+    assert not entities[0]["in_source"]
+    assert summary[entities[0]["start"] : entities[0]["end"]] == entities[0]["ent"]
+
+    assert entities[1]["ent"] == '"Brad"'
+    assert entities[1]["type"] == "PERSON"
+    assert not entities[1]["in_source"]
+    assert summary[entities[1]["start"] : entities[1]["end"]] == entities[1]["ent"]
+
+
+def test_split_person_dash():
```

Comment:
This is the test that would have failed without this behavior change, correct? The others tests are just backfilling?

Reply:
_dash, _apostrophe, _quote and _st were all unsupported prior to this PR, altho I had added manual handling for some of them without tests.

Reply:
Correct!